package/cloudshell/iac/terraform/services/svc_attribute_handler.py

```diff
@@ -12,3 +12,13 @@ def get_attribute(self, attribute_name: str) -> str:
         if f"{self._tf_service.cloudshell_model_name}.{attribute_name}" in self._attributes:
             return self._attributes[f"{self._tf_service.cloudshell_model_name}.{attribute_name}"]
         return ""
+
+    def check_attribute_exist(self, attribute_name: str) -> bool:
+        return attribute_name in self._tf_service.attributes or \
+               f"{self._tf_service.cloudshell_model_name}.{attribute_name}" in self._tf_service.attributes
```

Comment:
why not -
        return attribute_name in self._tf_service.attributes or check_2nd_gen_attribute_exist(attribute_name)

Reply:
correct